ui/pages/login.tsx

```diff
@@ -33,21 +33,16 @@ import {
 const Login: NextPage = () => {
   const [flow, setFlow] = useState<LoginFlow>();
   const [isSequencedLogin, setSequencedLogin] = useState(false);
-  const [baseURL, setBaseURL] = useState("");
   const isAuthCode = flow?.ui.nodes.find((node) => node.group === "totp");
 
   useEffect(() => {
-    void fetch("/api/v0/app-config")
+    void fetch("../api/v0/app-config")
       .then((response) => {
         const data = response.json();
-        return data as {
-          oidc_webauthn_sequencing_enabled?: boolean;
-          base_url?: string;
-        };
+        return data as { oidc_webauthn_sequencing_enabled?: boolean };
```

Comment:
suggestion: would it make sense to define an object with the `app-config` fields?

Reply:
added a type for this.